src/benchmark/Akka.Cluster.Benchmarks/Sharding/ShardingInfrastructure.cs

```diff
@@ -33,11 +33,11 @@ public ResolveResp(string entityId, Address addr)
             public Address Addr { get; }
         }
         
-        public ShardedEntityActor()
+        public ShardedEntityActor(string entityId)
         {
-            Receive<ShardingEnvelope>(e =>
+            Receive<Resolve>(_ =>
```

Comment:
This is the cause for the bug.

`ShardingEnvelope` does not get passed to the shard entity anymore, it is automatically unwrapped by the new extractor.

Reply:
BITTEN BY MY OWN BUG FIX

Reply:
Thank you @Arkatufus